tests/kernel/mutex/mutex_error_case/src/test_mutex_error.c

```diff
@@ -193,65 +112,106 @@ static int create_negative_test_thread(int choice)
 	return ret;
 }
 
-/* TESTPOINT: Pass a null pointer into the API k_mutex_init */
+
+/**
+ * @brief Test initializing mutex with a NULL pointer
+ *
+ * @details Pass a null pointer as a parameter then expected error happened.
+ *
+ * @ingroup kernel_mutex_tests
+ *
+ * @see k_mutex_init()
+ */
 static void test_mutex_init_null(void)
 {
 	create_negative_test_thread(MUTEX_INIT_NULL);
 }
-/* TESTPOINT: Pass a null pointer into the API k_mutex_lock */
-static void test_mutex_lock_null(void)
+
+/**
+ * @brief Test initialize mutex with a invalid kernel object
+ *
+ * @details Pass a invalid kobject as a parameter then expected error happened.
```

Comment:
Same as line 119.

Same problems at several lines. Please check/correct them all.

Reply:
Got it fixed, thanks!